homeassistant/components/atag/config_flow.py

```diff
@@ -36,7 +36,7 @@ async def async_step_user(self, user_input=None):
         except pyatag.errors.Unauthorized:
```

Comment:
Could this use `invalid_auth`?

Reply:
Good catch. 👍 

https://github.com/MatsNl/pyatag/blob/master/pyatag/gateway.py#L167-L169
https://github.com/MatsNl/pyatag/blob/master/pyatag/gateway.py#L74-L75